test/arkane/encorr/isodesmicTest.py

```diff
@@ -213,36 +226,20 @@ def test_calculating_constraints(self):
             "6_ring",
         }
 
-        # Now that we have confirmed that the correct keys are present, overwrite the constraint map to set the order
-        spcs_consts.constraint_map = {
-            "H": 0,
-            "C": 1,
-            "O": 2,
-            "N": 3,
-            "C=O": 4,
-            "C-N": 5,
-            "C-H": 6,
-            "C=C": 7,
-            "C=N": 8,
-            "C-C": 9,
-            "5_ring": 10,
-            "6_ring": 11,
-        }
-
         target_consts, consts_matrix = spcs_consts.calculate_constraints()
 
         # First, test that ethyne is not included in the reference set
         assert spcs_consts.reference_species == [self.propene, self.butane, self.benzene]
 
         # Then, test the output of the calculation
-        assert np.array_equal(target_consts, np.array([10, 8, 2, 4, 2, 10, 10, 1, 1, 1, 1, 1]))
+        assert np.array_equal(target_consts, np.array([ 1,  1,  1, 10, 10,  1,  1,  2,  0,  8, 10,  4,  2]))
         assert np.array_equal(
             consts_matrix,
             np.array(
                 [
-                    [6, 3, 0, 0, 0, 0, 6, 1, 0, 1, 0, 0],
-                    [10, 4, 0, 0, 0, 0, 10, 0, 0, 3, 0, 0],
-                    [6, 6, 0, 0, 0, 0, 6, 3, 0, 3, 0, 1],
+                    [ 0,  0,  1,  6,  0,  1,  0,  0,  0,  3,  6,  0,  0],
+                    [ 0,  0,  3, 10,  0,  0,  0,  0,  0,  4, 10,  0,  0],
+                    [ 0,  1,  3,  6,  0,  3,  0,  0,  0,  6,  6,  0,  0],
```

Comment:
I suggest adding the results of `[feat.__repr__() for feat in caffeine_features]` as a comment in this section, to indicate what these numbers are?

Reply:
Thanks, done